src/Maestro/Maestro.Web/Startup.cs

```diff
@@ -370,7 +370,6 @@ private void ConfigureCookieAuthedApi(IApplicationBuilder app)
                 app.MapWhen(ctx => ctx.Request.Cookies.TryGetValue("Skip-Api-Redirect", out _),
                     a =>
                     {
-                        a.UseRouting();
```

Comment:
Is this ok?

Reply:
yes, the AddRouting is higher in this method.